modules/assim.sequential/inst/restart_SDAworkflow_scripts/SDA_Workflow_LAI.R

```diff
@@ -0,0 +1,361 @@
+# ----------------------------------------------------------------------
+#------------------------------------------ Load required libraries-----
+# ----------------------------------------------------------------------
+library("PEcAn.all")
+library("PEcAn.utils")
+library("PEcAn.data.remote")
+library("PEcAnAssimSequential")
+library("RCurl")
```

Comment:
You can remove this line -- nothing in PEcAn uses RCurl anymore 🎉

Reply:
Roger that!